amethyst_assets/examples/hl.rs

```diff
@@ -36,6 +37,7 @@ impl App {
         world.add_resource(Errors::new());
         world.add_resource(AssetStorage::<MeshAsset>::new());
         world.add_resource(Loader::new(path, pool.clone()));
+        world.add_resource(Time::placeholder());
```

Comment:
Could this use `Time::default` instead of creating a new `Time::placeholder` method?

Reply:
Much better, I didn't see the `Default` implementation.